spec/repository_spec.rb

```diff
@@ -26,3 +26,17 @@
     expect(EXTRA_FILES).to eq([])
   end
 end
+
+describe "The file \"renamed-styles.json\"" do
+  it "must be valid JSON" do
+    expect(JSON_IS_VALID).to be true
+  end
```

Comment:
> but basically each style is read and parsed (by nokogiri) twice

Could I check how much time is spent on this by timing `CSL::Style.load(path)` (read/parse) vs. `CSL.validate(path)` (read/parse/validate)?

By the way, the profiling output is:

```
Top 10 slowest examples (9.98 seconds, 13.4% of total time):
  zeitschrift-fur-soziologie: must validate against the CSL 1.0.1 schema
     Please check your style at http://validator.citationstyles.org/
    1.68 seconds ./spec/styles_spec.rb:4
  zeitschrift-fur-soziologie: must be parsable as a CSL style
    1.6 seconds ./spec/styles_spec.rb:9
  pontifical-gregorian-university: must be parsable as a CSL style
    1.03 seconds ./spec/styles_spec.rb:9
  pontifical-gregorian-university: must validate against the CSL 1.0.1 schema
     Please check your style at http://validator.citationstyles.org/
    1.02 seconds ./spec/styles_spec.rb:4
  dependent/nigerian-journal-of-pharmaceutical-sciences: may not have <macro/>,
<citation/>, or <bibliography/> elements
    0.79808 seconds ./spec/styles_spec.rb:94
  dependent/remote-sensing-applications-society-and-environment: must have an <i
nfo/> element
    0.79308 seconds ./spec/styles_spec.rb:116
  dependent/the-journal-of-heart-and-lung-transplantation: must have a style ID
    0.78908 seconds ./spec/styles_spec.rb:120
  dependent/journalism-practice: must validate against the CSL 1.0.1 schema
     Please check your style at http://validator.citationstyles.org/
    0.76408 seconds ./spec/styles_spec.rb:4
  dependent/journal-of-hydrometeorology: may not have <macro/>, <citation/>, or
<bibliography/> elements
    0.75808 seconds ./spec/styles_spec.rb:94
  dependent/international-journal-of-surgery-open: must validate against the CSL
 1.0.1 schema
     Please check your style at http://validator.citationstyles.org/
    0.75508 seconds ./spec/styles_spec.rb:4

Top 10 slowest example groups:
  zeitschrift-fur-soziologie:
    0.1939 seconds average (3.3 seconds / 17 examples) ./spec/styles_spec.rb:155

  pontifical-gregorian-university:
    0.12107 seconds average (2.06 seconds / 17 examples) ./spec/styles_spec.rb:1
55
  soziologie:
    0.05136 seconds average (0.87309 seconds / 17 examples) ./spec/styles_spec.r
b:155
  dependent/nigerian-journal-of-pharmaceutical-sciences:
    0.05038 seconds average (0.80608 seconds / 16 examples) ./spec/styles_spec.r
b:162
  dependent/remote-sensing-applications-society-and-environment:
    0.05001 seconds average (0.80008 seconds / 16 examples) ./spec/styles_spec.r
b:162
  dependent/the-journal-of-heart-and-lung-transplantation:
    0.04988 seconds average (0.79808 seconds / 16 examples) ./spec/styles_spec.r
b:162
  dependent/journalism-practice:
    0.04825 seconds average (0.77208 seconds / 16 examples) ./spec/styles_spec.r
b:162
  dependent/journal-of-hydrometeorology:
    0.04782 seconds average (0.76508 seconds / 16 examples) ./spec/styles_spec.r
b:162
  dependent/hormones-and-cancer:
    0.04769 seconds average (0.76308 seconds / 16 examples) ./spec/styles_spec.r
b:162
  dependent/international-journal-of-surgery-open:
    0.04763 seconds average (0.76208 seconds / 16 examples) ./spec/styles_spec.r
b:162
```

(I'm not sure what makes https://github.com/citation-style-language/styles/blob/master/zeitschrift-fur-soziologie.csl so slow; I've seen this before, and the style isn't particularly complex)